_clanky/výběr-katalogizačního-systému-pro-město-Brno.md

```diff
@@ -1,178 +1,83 @@
 
-Co dodat na závěr? Výběr vhodného katalogizačního systému je náročnou disciplínou. Častokrát je těžké si správně vybrat bez předešlých zkušeností a detailní znalosti problematiky  Někdy se stane, že si samospráva (jako například v případě Brna) nevybere pro svoje potřeby nejvhodnější technologii. Tímto samozřejmě **netvrdíme**, že CKAN nebo jiný systém je horší než systém HUB. Při výběru je totižto nutné vzít v potaz technologická, personální nebo finanční specifika každého města a ta pak určují, jaký systém je nejvhodnější. ArcGIS HUB například **určitě není vhodným řešením** pro organizace, které nemají podstatnou část svých dat ve službách nebo geodatabázích postavených nad architekturou od ESRI.
+Co dodat na závěr? Výběr vhodného katalogizačního systému je náročnou disciplínou. Častokrát je těžké si správně vybrat bez předešlých zkušeností a detailní znalosti problematiky  Někdy se stane, že si samospráva (jako například v případě Brna) napoprvé nevybere pro svoje potřeby nejvhodnější technologii. Tímto samozřejmě **netvrdíme**, že CKAN nebo jiný systém je horší než systém HUB. Při výběru je totižto nutné vzít v potaz technologická, personální nebo finanční specifika každého města a ta pak určují, jaký systém je nejvhodnější. ArcGIS HUB například **určitě není vhodným řešením** pro organizace, které nemají podstatnou část svých dat ve službách nebo geodatabázích postavených nad architekturou od ESRI.
```

Comment:
@spalrobert totižto jsem vygooglil jen v sk, česky to neni :D

Reply:
opravené :)